### PR TITLE
Eq & Show instances for StreamingClientStatus.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: core/shared/src/main/scala/clue/StreamingClientStatus.scala [hunk 1]
```diff
@@ -3,10 +3,16 @@
 
 package clue
 
+import cats.Eq
+import cats.Show
+
 sealed trait StreamingClientStatus
```
Comment (cquiroz) [inline core/shared/src/main/scala/clue/StreamingClientStatus.scala:9]: Should this be an `Enumerated`?

Comment (rpiaggio) [reply core/shared/src/main/scala/clue/StreamingClientStatus.scala:9]: I'm not entirely convinced of introducing a dependency on `lucuma-core` for this.

Comment (cquiroz) [reply core/shared/src/main/scala/clue/StreamingClientStatus.scala:9]: Oh, I thought `lucuma-core` was there already. I agree it is not needed in that case

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**